cves/CVE-2016-3092.yml

```diff
@@ -43,37 +43,32 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The system doesn't check the length of a boundary string. This means that
```

Comment:
What part of the system doesn't check for the boundary length?

Reply:
It's the multipart stream part of the system. I will add that to the description.